src/qibo/tests_new/test_abstract_circuit_qasm.py

```diff
@@ -346,7 +347,9 @@ def test_from_qasm_invalid_script():
         c = Circuit.from_qasm(target)
 
 
-def test_from_qasm_measurements():
+def test_from_qasm_measurements(backend):
```

Comment:
I do not mind with testing the different backends here, just noting that my initial idea of `test_abstract_*` was to be completely backend free and just test the basic functionality.

Reply:
Right, but here (and in the other places) I am forced to pass backend, otherwise when you set `--engines numpy` and you have tf installed, it will use `defaulteinsum` from tensorflow instead of numpy.